ibazel/command/notify_command.go

```diff
@@ -45,20 +47,26 @@ func NotifyCommand(startupArgs []string, bazelArgs []string, target string, args
 }
 
 func (c *notifyCommand) Terminate() {
-	if c.pg == nil || !subprocessRunning(c.pg.RootProcess()) {
+	if !c.IsSubprocessRunning() {
```

Comment:
I think a [`sync.Once`](https://golang.org/pkg/sync/#Once) would be an effective strategy here as well